audio-worklet/basic/message-port.html

```diff
@@ -15,11 +15,17 @@ <h1>
         MessagePort with AudioWorklet
       </h1>
       <p>
-        Demonstrates the basic bi-directionaly communication between
-        AudioWorkletNode and AudioWorkletProcessor.
+        Demonstrates basic bi-directional communication between AudioWorkletNode
+        and AudioWorkletProcessor.
       </p>
       <p>
         <strong>NOTE</strong>: Open up the console to see messages between them.
+        <ul>
+          <li>- Upon the construction of <code>PortWorkletNode</code>, it will
```

Comment:
Ditti about "-".

Reply:
Ditto. If this is a problem, I can rewrite the CSS for the page as a separate PR.